ui/src/CollectionExerciseDetails.js

```diff
@@ -613,6 +613,12 @@ class CollectionExerciseDetails extends Component {
     );
 
     const actionRuleTableRows = sortedActionRules.map((actionRule, index) => {
+      const updatedDateTime = new Date(
```

Comment:
Perhaps better to name this something like `localisedDateTime` to avoid confusion with the mechanism to update the trigger time?

Reply:
Good spot. Was meaning to change that name